src/data/global.json

```diff
@@ -4,6 +4,10 @@
         "name": "favicon",
         "include": "all"
     },
+    "banner": {
+        "en": "The CWA development ends on May 31, 2023. You still can warn other users until April 30, 2023. More information: <a href='/en/faq/results/#ramp_down' target='_blank'>FAQ</a>",
+        "de": "Die CWA-Entwicklung endet am 31.05.2023. Bis einschließlich 30.04.2023 können Sie andere Anwender warnen. Mehr dazu hier: <a href='/de/faq/results/#ramp_down' target='_blank'>FAQ</a>"
```

Comment:
```suggestion
        "de": "Die CWA-Entwicklung endet am 31.05.2023. Bis einschließlich 30.04.2023 können Sie andere warnen. Mehr dazu hier: <a href='/de/faq/results/#ramp_down' target='_blank'>FAQ</a>"
```

Reply:
@Ein-Tim 
Please check if "andere" should be capitalized as "Andere".

Reply:
@MikeMcC399 I'm unsure too! I will look at it again.

Reply:
I think a small i is correct here, see this screenshot from the app itself: 
![IMG_74D7B378F5FB-1](https://user-images.githubusercontent.com/67682506/227470363-5dd5451d-ab37-4359-b78b-770e7747dcfd.jpeg)

Reply:
@Ein-Tim 

In the legal texts it is written capitalized:

"Ihre Daten werden im Zusammenhang mit den Funktionen der App grundsätzlich nur auf Grundlage Ihres Einverständnisses verarbeitet. Dies umfasst z. B. die Datenverarbeitung zur Risiko-Ermittlung und der Warnung **Anderer**, die Überprüfung von COVID-Zertifikaten bei Ticketbuchungen sowie die Verarbeitung von Daten in COVID-Zertifikaten bei der Aktualisierung von Zertifikaten (siehe Punkt 6.o)."

Reply:
Okay, I'll leave it to the team to decide here :D

Reply:
In this sentence "andere" is not capitalized because it is not a substantivized term. "Andere" is used as an adjective here and describes the noun "Anwender". E.g. "Warnen sie andere (Teilnehmer)".

I actually think the example @MikeMcC399 refers to is wrong. Its not impossible for "Anderer" to be uppercased, but that would mean "Anderer" is a part of Warnung rather than some noun like Teilnehmer e.g. "... unde der Warnung anderer Teilnehmer".

Reply:
Well, then again. Words may be capitalized, if they are used as a "Substantiv". So this might be entirely correct.

Reply:
https://mentor.duden.de suggest using a small case i. Thus, I'm suggesting the same! 🙈

Reply:
Sorry for the confusion. Apparently the legal text does not conform to the recommendations from Duden.

![image](https://user-images.githubusercontent.com/66998419/227490461-f8400df0-b8b5-4f6c-b4db-c82be0a0dc9e.png)

Reply:
https://www.duden.de/sprachwissen/sprachratgeber/Die-Schreibung-von-%E2%80%9Eandere%E2%80%9C%E2%80%9Eanderer%E2%80%9C%E2%80%9Eanderes%E2%80%9C
"Großschreiben kann man allerdings (muss es aber nicht), wenn das Wort nicht als unbestimmtes Pronomen zu verstehen ist, sondern eine ganz bestimmte Vorstellung auslösen soll: die Suche nach dem Anderen (= nach einer neuen, unbekannten Welt) oder der Dialog mit dem Anderen (= dem Mitmenschen, dem Gegenüber). Auf der verzweifelten Suche nach dem Anderen hat sie ihr eigentliches Ziel gänzlich aus den Augen verloren."

I suppose its correct and there is no need to change the legal texts. I'd also go with the lowercase though.

Reply:
Back to the original suggestion:

The texts are coming from SAP directly and we paid close attention to having them all be the same across the different services (App, website, ...). Also this would create a slight inconsistency:
"können Sie andere warnen" translates directly to: "**others** can be warned" rather than "**other users** can be warned" liek in the texts

Reply:
IMO it's redundant to say "users" / "Anwender", because it is clear that only these people can be warned via the app. But if these texts have already been approved by SAP, please go ahead and leave them like they are!

Reply:
Thanks for your suggestion anyway! I'll close this conversation now.